packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -131,7 +135,7 @@ export default class Editor implements IEditor {
             this.core.plugins[i].dispose();
         }
 
-        this.core = null;
+        this.core = null!;

```

Comment:
Actually, this.core can really be null, so we should allow it when declare the type.

Then in each place we use it, we can check and throw:
```ts
const Disposed = 'Editor is already disposed';

...

if (!this.core) { throw newError(Disposed); }

... // now you can use this.core
}
```

Reply:
Hm, checking on each method may be a little too WET
![image](https://user-images.githubusercontent.com/44042957/188006928-2924cd65-0db9-4567-bdc2-ad7531f7b3ce.png)
I'm wondering if I should introduce a private `getCore` and run this check there, replacing `this.core` reads with `this.getCore()`

Reply:
A private getCore() sounds good to me